gobblin-test-harness/src/main/java/gobblin/test/execution/operator/Operator.java

```diff
@@ -0,0 +1,18 @@
+package gobblin.test.execution.operator;
+
+
+/**
+ * An interface for defining the operator , the operator could be a copy of file or converting a file from one format to another
+ * 
+ * @author sveerama
+ */
+
+public interface Operator
```

Comment:
Are these operators only going to be used in the set up phase? If so then it may make more sense to call this SetupOperator